### PR TITLE
OV-307: Show Preview Url Of Each Video as Thumbnails

### LINKED ISSUE #307: BUG: Generic Thumbnails Displayed Instead of Video Thumbnails
### Describe your issue

When displaying the user's videos on the home page, the system generates and shows a generic placeholder thumbnail for each video. The thumbnails are not correctly representing the actual content of the uploaded videos.

### Steps to reproduce

1. Navigate to the home page as a logged-in user with uploaded videos.
2. Observe the video list/grid displaying the user's videos.
3. Check the thumbnail for each video in the list.

### What was the actual result?

All videos are shown with the same generic placeholder thumbnail instead of a video-specific thumbnail.

Image

### What was the expected result?

Each video should have a thumbnail that accurately represents a frame from the video itself 

### Put here any screenshots or videos (optional)

_No response_

### What OS are you seeing the problem on?

Windows

### What browsers are you seeing the problem on?

Chrome, Safari, Edge, Firefox, Opera

### Severity

Minor (S4)

### PR DESCRIPTION
**Pull request by lfelix3011**
This PR is for this Task https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/issues/307

image

### CODE REVIEW

**Comment by Sanchousina** [comment]
I already did that when working on the task: https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/issues/311

**Review by Alblupynos — DISMISSED** [review]

**Review by stefano-lacorazza — DISMISSED** [review]

File: frontend/src/bundles/home/components/video-card/video-card.tsx [hunk 1]
```diff
@@ -63,7 +64,11 @@ const VideoCard: React.FC<Properties> = ({ id, name, url }) => {
     return (
         <Box borderRadius="8px" bg="white" padding="7px">
             <Box position="relative" role="group">
-                <Image src={photo} alt="Video preview" borderRadius="5px" />
+                <Image
+                    src={previewUrl || photo}
```
Comment (anton-otroshchenko) [inline frontend/src/bundles/home/components/video-card/video-card.tsx]: please replace || with ??

**Review by anton-otroshchenko — CHANGES_REQUESTED** [review]

**Comment by Sanchousina** [comment]
Look at this commit: https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/343/commits/24d5376bf3581d92c238aaa6ab29ecc0684e0a0e

**Comment by lfelix3011** [comment]
The changes made here were also done in this PR https://github.com/BinaryStudioAcademy/bsa-2024-outreachvids/pull/343m
, as that one was already merge, this one will be closed

**Pull request closed without merging**